exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = htmlLevel > 5;
+let cssLevelAbove5 = cssLevel > 5;
```

Comment:
Hi Jude, have a look at this one, it seems like it's returning false, instead of true. Also good work on your code, keep it up!

Reply:
Amazing, thank you, I'll take a look.